File: src/chainOfResponsibility/java/com/my/UserExistsCheckLink.java
package src.chainOfResponsibility.java.com.my;

public class UserExistsCheckLink extends ChainLink {
    @Override
    public boolean check(String username, String password, String token) {
        if (username.equals("admin") && password.equals("admin")) {
            return next(username, password, token);
        } else if (username.equals("1") && password.equals("1")) {
            return next(username, password, token);
        }

        return false;
    }
}

File: src/chainOfResponsibility/python/UserExistsCheckLink.py
from src.chainOfResponsibility.python.ChainLink import ChainLink


class UserExistsCheckLink(ChainLink):
    def check(self, username: str, password: str, token: str) -> bool:
        if username == "admin" and password == "admin":
            return self.next(username, password, token)
        elif username == "1" and password == "1":
            return self.next(username, password, token)
        return False
